### DISCUSSION TITLE
"HTTP request with no content length" after upgrade

### DISCUSSION THREAD

**Discussion by ptacek** [ClickHouse/ClickHouse#48096] [Q&A] [answered]
Hi,
i've upgraded from CH 22.7. to 23.2. After the upgrade, I noticed these warning messages in logs:
`<Warning> HTTPServerRequest: Got an HTTP request with no content length and no chunked/multipart encoding, it may be impossible to distinguish graceful EOF from abnormal connection loss`

Do you have any idea what might be causing it? It's like 20 messages per second, so it's quite flooding the log.
Thanks for any help.

**Comment by Algunenano** [comment, marked as answer]
Should be fixed by https://github.com/ClickHouse/ClickHouse/pull/47903 (for older releases) and https://github.com/ClickHouse/ClickHouse/pull/47950 for 23.3+

**Reply by ptacek** [reply]
Thank you for your answer. I will upgrade to 23.3. when it's released and write here whether it has stopped.
